lib/rules/color-contrast-matches.js

```diff
@@ -10,6 +10,30 @@ if (
 	return false;
 }
 
+// form elements that don't have direct child text nodes need to check that
+// the text indent has not been changed and moved the text away from the
+// control
+const formElements = ['INPUT', 'SELECT', 'TEXTAREA'];
+const style = window.getComputedStyle(node);
```

Comment:
Minor point, but I'm thinking we should only call getComputedStyle after we've tested nodeName, to minimise the number of style computations we're doing here.